src/test/java/org/jenkinsci/plugins/github/internal/GitHubClientCacheCleanupTest.java

```diff
@@ -103,7 +102,10 @@ private void it(String comment, int count) throws IOException {
     }
 
     private void makeCachedRequestWithCredsId(String credsId) throws IOException {
-        jRule.getInstance().getDescriptorByType(GitHubServerConfig.DescriptorImpl.class)
-                .doVerifyCredentials(github.serverConfig().getApiUrl(), credsId, 1);
+        GitHubServerConfig config = new GitHubServerConfig(credsId);
+        config.setApiUrl(github.serverConfig().getApiUrl());
+        config.setClientCacheSize(1);
```

Comment:
@kohsuke maybe it would be easy for you to implement this https://issues.jenkins-ci.org/browse/JENKINS-32503 ?